cmd/internal/cli/startvm_darwin.go

```diff
@@ -70,16 +69,31 @@ func getHypervisorArgs(sifImage, bzImage, initramfs, singAction, cliExtra string
 		sylog.Fatalf("Failed to get current user")
 	}
 
+	// NOTE: The 0:4:x PCI slot is to be used for static mounts. (BUS:SLOT:FUNCTION)
 	// Force $HOME to be mounted
 	// TODO: engineConfig.GetHomeSource() / GetHomeDest() -- should probably be used
 	homeSrc := usr.HomeDir
 	pciArgs := fmt.Sprintf("4:0,virtio-9p,home=%s", homeSrc)
 	homeBind := fmt.Sprintf("home:%s", homeSrc)
 	singBinds = append(singBinds, homeBind)
-
 	sylog.Debugf("PCI: %s", pciArgs)
 	args = append(args, "-s", pciArgs)
 
+	// Check for Sandbox Image
+	sylog.Debugf("Check for sandbox image")
+	if f, err := os.Stat(sifImage); err == nil {
+		if f.IsDir() {
+			sylog.Debugf("Image is sandbox. Setting up share.")
+			pciArgs = fmt.Sprintf("4:1,virtio-9p,runimg=%s", sifImage)
+			args = append(args, "-s", pciArgs)
+			sboxImgBind := fmt.Sprintf("runimg:/runImage")
```

Comment:
`runImage` or sifImage value ?

Reply:
`runimg` is the 9p mount tag. `/runImage` is the mount point.

Reply:
To expand on this... line 87, it sets:
`virtio-9p,runimg=[sifImage]`

Where `runimg` is the 9p mount tag, and what we're passing to be mounted is the value of `sifImage`, which in this case is a sandbox directory.